src/SourceBuild/patches/runtime/0001-PATCH-Only-update-runtime-maintenance-packages-depen.patch

```diff
@@ -0,0 +1,78 @@
+From bdada66846ba8164466d0e5b00bb14d12f194bcd Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Carlos=20S=C3=A1nchez=20L=C3=B3pez?=
+ <1175054+carlossanlop@users.noreply.github.com>
+Date: Mon, 25 Nov 2024 16:56:39 -0800
+Subject: [PATCH] [PATCH] Only update runtime maintenance-packages dependencies
+ to latest when not in source-build.
+
+Without this workaround, the deps flow from runtime to sdk will get blocked by roslyn because that
+repo will downgrade the package version.
+
+Roslyn cannot immediately get these same dependencies updated because msbuild will fail to find them.
+
+MSBuild needs to get these package versions updated before all other repos.
```

Comment:
Per the [patch guidelines](https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines), please add a `Backport:<issue/PR>` line to track the work that will remove this patch.

Reply:
I created https://github.com/dotnet/sdk/issues/45155 to track this.